### ISSUE TITLE
afebrile seizures

### ISSUE THREAD

**Issue by uisch** [obophenotype/human-phenotype-ontology#3776]
_For new term requests, please provide us with the following information:_
### 1. Preferred term label
afebrile seizures

### 2. Synonyms
afebrile convulsions; seizures, not associated with fever

### 3. Textual definition (should be understandable even for non-specialists, please include a PubMed ID for relevant articles providing additional information about the suggestion)
afebrile seizures are convulsions that are not associated with fever

### 4. Parent term (use [HPO Browser](http://compbio.charite.de/hpoweb/showterm?id=HP:0000118) or [OLS](http://www.ebi.ac.uk/ols/ontologies/hp))
 HP:0001250

### 5. Which diseases are characterized by this term ? (e.g. Orphanet- or OMIM-id)
N/A

### 6. Your nano-attribution (ORCID-id or label, e.g. _HPO:probinson_ (organization:name))
N/A

**Comment by drseb** [comment]
How does this relate to https://hpo.jax.org/app/browse/term/HP:0040168 ?

**Comment by uisch** [comment]
https://hpo.jax.org/app/browse/term/HP:0040168 by definition is focal, whereas afebrile seizures could be generalized as well

**Comment by drseb** [comment]
Ok. Thanks

**Comment by pnrobinson** [comment]
I do not think this is a good term, because essentially all of the seizure terms except a few are not febrile. I think that you are referring to the situation when a young child has initial seizures (https://www.bcmj.org/articles/approach-child-first-afebrile-seizure) and we want to say that the seizures are NOT febrile seizures? In this case, the "afebrile" is refering to the etiology (i.e., it is ruling out febrile seizures as the diagnosis), but it is not specifying the phenotype, because it could be any of a number of seizure semiologies. I will close this item for now, but please reopen it if I am misunderstanding something.
